GOVERNANCE.md

```diff
@@ -0,0 +1,40 @@
+# Project Governance
+
+The development and community management of the project will follow the governance rules described in this document.
+
+## Project Maintainers
+
+Project maintainers have admin access to the GitHub repository. The team of project maintainers is the following:
+
+* [José Antonio Hernández](https://github.com/jlcanovas/) (he/him)
+* [Javier Cánovas](https://github.com/jlcanovas/) (he/him)
+* [Jesús Sánchez](https://github.com/jlcanovas/) (he/him)
+
+## Project Collaborators & Contributors
+
+Any member willing to participate in the development of the project will be considered as a collaborator. 
+
+Collaborators may propose changes to the project's source code. The mechanism to propose such a change is a GitHub pull request. A collaborator proposing a pull request is considered a contributor. Project maintainers will review and merge pull requests.
+
+## Issue governance
+
+Both collaborators and project maintainers may propose issues. The participation in the issue discussion is open and must follow the [Code of Conduct](CODE_OF_CONDUCT.md).
+
+The group of project maintainers will be responsible for assigning labels to issues, as well as assign the issue to a project maintainer or contributor.
+
+The group of project maintainers commit to give an answer to any issue in a period of time of **48 hours**. 
```

Comment:
As PR timeframe has been changed, @jesusc do you think we shuold also update the time for issues?

Reply:
I think answering an issue is ok like this.